apps/jobs/views.py

```diff
@@ -539,3 +540,38 @@ def users(self, request):
         return response.create_response(
             serialized_company_data.data, status.HTTP_200_OK
         )
+
+
+class ContactUsViewSet(viewsets.ModelViewSet):
+    """Company object viewsets
+    API: /api/v/contact-us
+    Database: jobs_contactmessage
+    Functions:
+        1. take input from user end
+        2. create contact message
+        3. contact message stored in database
+    """
+
+    # queryset = ContactMessage.objects.all()
+    serializer_class = ContactUsSerializer
+
+    def get_queryset(self):
+        return ContactMessage.objects.all()
+
+    @action(detail=False, methods=["post"])
+    def create_contact_message(self, request):
+        full_name = request.data.get("full_name")
+        email = request.data.get("email")
+        message = request.data.get("message")
+
+        validation_error = validationClass.validate_fields(full_name, email, message)
+
+        if validation_error:
+            return Response(validation_error, status=status.HTTP_400_BAD_REQUEST)
+
+        serializer_class = ContactUsSerializer(data=request.data)
+        if serializer_class.is_valid():
+            serializer_class.save()
+            return Response(serializer_class.data, status=status.HTTP_200_OK)
+        else:
+            return Response(serializer_class.errors, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
Ensure such exceptions don't expose sensitive details such as the id of user, stack trace etc.